Grbl_Esp32/src/Probe.cpp

```diff
@@ -24,47 +24,40 @@
 #include "Grbl.h"
 
 // Inverts the probe pin state depending on user settings and probing cycle mode.
-uint8_t probe_invert_mask;
+bool is_probe_away;
```

Comment:
This should be static since it is only used within this file and set via a setter function